Pipfile

```diff
@@ -4,12 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [dev-packages]
-flake8 = "*"
-pylint = "*"
```

Comment:
Just to be sure, did you intend to remove these two?  ^

Reply:
Yes, those are tools that I like to have in my development environment that not every one else does. I did add a flake8 test to tox and travis to catch issues.